### PR TITLE
map choropleth (#507)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
Add support for choropleth Maps

### CODE REVIEW

File: gui/src/components/Taipy/Chart.tsx [hunk 1]
```diff
@@ -62,6 +64,9 @@ interface ChartProp extends TaipyActiveProps, TaipyChangeProps {
     render?: boolean;
     defaultRender?: boolean;
     decimator?: string;
+    template?: string;
```
Comment (dinhlongviolin1) [inline gui/src/components/Taipy/Chart.tsx]: same based branch as the chart theme pr I guess

**Review by dinhlongviolin1 — APPROVED** [review]
🚀

File: gui/src/components/Taipy/Chart.tsx (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/src/components/Taipy/Chart.tsx]: Indeed, yes, I'll merge it after the theme one

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**